app/views/community/index.html.slim

```diff
@@ -31,8 +31,7 @@ table.users.table.table-striped.table-bordered.table-condensed.table-hover
         td.image = avatar_url(user, size: 40)
         td = link_to user.name_or_handle, user
         td = user.interested_in.map { |key| User::INTERESTS[key].presence }.compact.join(', ')
-        - if signed_in?
-          td = mail_to user.email unless user.hide_email?
+        td = mail_to user.email if can?(:read_email, user)
```

Comment:
🙈 I thought I had it right. Sorry. 
I didn't miss your suggestion, I just found that it gave problems when a user as their email hidden. Look:

![two_line_if_one_hidden_email](https://user-images.githubusercontent.com/6314687/40625145-f72b3744-62b0-11e8-86eb-252e3f1a22df.png)
(first user hides email).

I will look again!!

Reply:
Ah, that's a great point - I'm sorry I missed that!

I haven't tested this, but maybe something like this would work:

```slim
- if can?(:read_email, nil)
  td = mail_to user.email if can?(:read_email, user)
```

The idea being that the first check will be `false` for unconfirmed / logged out users, while being true for other users. It's not the prettiest thing every, but it may solve the issue without us needing to explicitly check if the user is confirmed (which duplicates knowledge that unconfirmed/logged out users should not see emails).